src/components/screens/vacation-requests-screen.tsx

```diff
@@ -287,7 +287,8 @@ const VacationRequestsScreen = () => {
           message: vacationData.message,
           createdAt: new Date(),
           updatedAt: new Date(),
-          days: vacationData.days
+          days: vacationData.days,
+          draft: false
```

Comment:
Why was this change added, is it essential?

Reply:
Remove the change and made it back to way it was